packages/tools/package/src/commands/changelog/changelog.ts

```diff
@@ -0,0 +1,54 @@
+import path from 'path';
+
+import { CommandsCommand } from '@webex/cli-tools';
+
+import { Package } from '../../models';
+import { Yarn } from '../../utils';
+import type { PackageConfig } from '../../models';
+
+import CONSTANTS from './changelog.constants';
+import type { Options } from './changelog.types';
+import { createOrUpdateChangelog } from './changelog.utils';
+
+/**
+ * @public
```

Comment:
We can add description here

Reply:
I've followed the style with which comments are written in the tooling. The description is written inside the constant so here line 17-20 has the description